src/main/java/net/earthcomputer/clientcommands/c2c/PacketCacheHelper.java

```diff
@@ -0,0 +1,24 @@
+package net.earthcomputer.clientcommands.c2c;
+
+import com.google.common.cache.CacheBuilder;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Set;
+
+public class PacketCacheHelper {
+
+    private static final Set<String> cache = Collections.newSetFromMap(CacheBuilder.newBuilder().expireAfterWrite(Duration.ofMinutes(1)).<String, Boolean>build().asMap());
+
+    public static boolean contains(String packetString) {
```

Comment:
This should probably be renamed to indicate that it's removing something. Currently the method name indicates it's an immutable query, and might accidentally be used as such.